shared/src/bundles/users/validation-schemas/user-sign-up.validation-schema.ts

```diff
@@ -6,19 +6,34 @@ import { type UserSignUpRequestDto } from '../types/types.js';
 const userSignUp = joi.object<UserSignUpRequestDto, true>({
     email: joi
         .string()
-        .trim()
+        .pattern(
+            new RegExp(
+                '^[a-zA-Z0-9._%+-]{1,15}@(?=.{3,15}$)[a-zA-Z0-9.-]{1,}\\.[a-zA-Z]{1,}$',
```

Comment:
What is this for?
Even if you need it, it's better to assign regular expressions to a variable